esp/src/src-react/components/LandingZone.tsx

```diff
@@ -136,6 +138,13 @@ export const LandingZone: React.FunctionComponent<LandingZoneProps> = ({
             displayName: tree({
                 label: nlsHPCC.Name,
                 sortable: false,
+                shouldExpand: function (row, level) {
+                    if ((dzExpanded === "" || dzExpanded === row.data.DropZone?.Name) && level <= 1) {
```

Comment:
@GordonSmith this was one of the issues I mentioned in our call this morning. This PR is backporting the changes for 29400 (opening the first LZ by default). And then actual fix which conflicted between 9.0.x & 9.2+ was checking for an optional `?.Name` here & passing the filter's selected dropzone into the store constructor (changed on line 118 above)